src/actions/setting/DefaultLanguage.js

```diff
@@ -0,0 +1,10 @@
+import { DEFAULT_LANGUAGE } from '../../types';
+
+export const defaultLanguage = ({ newLang }) => (dispatch) => {
```

Comment:
`defaultLanguage` is not a verb and does not describe the action correctly. Should be `changeLanguage`.